### ISSUE TITLE
Placeholders support

### ISSUE THREAD

**Issue by Lightwav3** [Kikisito/NFCNotes#15]
Hey, again. I just wanna ask if placeholders are supported for this plugin and, I was planning to add like the placeholder for the money. Or.. you could add it yourself?

Example:

Before: $1000
After: $1k

The placeholder is called:
%nf_###E0X_{placeholder}%

Now, I probably don't remember if your plugin already supports this. But I'm just double checking lol. Correct me if i'm wrong.

**Comment by Kikisito** [comment]
NFCNotes doesn't provide any placeholder. If you think any specific placeholder may be useful, please tell me.

Regarding placeholders in the notes' name and/or lore, these are not supported at the moment but I think it's a great idea to have them. I'll definitely add them soon.

**Comment by Kikisito** [comment]
I guess the placeholder you are using as example is a placeholder provided by PlaceholderAPI. Am I wrong?

**Comment by Lightwav3** [comment]
> I guess the placeholder you are using as example is a placeholder provided by PlaceholderAPI. Am I wrong?

Yes, it is.

**Comment by Kikisito** [comment]
I don't think this will be added, at least soon, as it isn't really useful. I'm sorry. I don't really have enough time to do it. In the future, if more users ask for it, placeholders can be reconsidered.
